services/concierge/actors/src/main/java/org/eclipse/ditto/services/concierge/actors/cleanup/EventSnapshotCleanupCoordinator.java

```diff
@@ -360,8 +362,10 @@ private void scheduleWakeUp(final Duration when) {
                         .match(ConnectionTag.class, connTag ->
                                 askShardRegionForCleanup(shardRegions.connections(), ConnectivityCommand.RESOURCE_TYPE, connTag))
                         .matchAny(e -> {
+                            final String msg = "Unexpected entity ID type: " + e.getClass().getSimpleName();
                             final CleanupPersistenceResponse error =
-                                    CleanupPersistenceResponse.failure("Unexpected entity ID type: ", DittoHeaders.empty());
+                                    //TODO: Dont build entity id of message.
```

Comment:
What about this TODO? Still open?

Reply:
Yes this todo is still open. I "worked around" this right now by wrapping the mssage in a "DefaultEntityId" but this is obviously not the way we should use this ID. I'd suggest opening a new issue to provide a proper way for error responses to CleanupPersistence commands. Is this okay for you?

Reply:
Yes, ok for me

Reply:
https://github.com/eclipse/ditto/issues/476